scenario/deletedata.go

```diff
@@ -7,20 +7,40 @@ import (
 	"github.com/pkg/errors"
 	"github.com/qlik-oss/gopherciser/action"
 	"github.com/qlik-oss/gopherciser/connection"
-	"github.com/qlik-oss/gopherciser/elasticstructs"
+	"github.com/qlik-oss/gopherciser/helpers"
 	"github.com/qlik-oss/gopherciser/logger"
 	"github.com/qlik-oss/gopherciser/session"
 )
 
 type (
+	// DeleteDataSettingsCore settings core used by UnmarshalJSON
+	DeleteDataSettingsCore struct {
+		Filename string `json:"filename" displayname:"Filename" doc-key:"deletedata.filename"`
+		SpaceID  string `json:"spaceid" displayname:"Space ID" doc-key:"deletedata.spaceid"`
```

Comment:
Maybe more as an improvement, but perhaps we should consider creating a SpaceSelection type so that the user may use names or IDs for specifying the space in a portable way.

Reply:
yea but that should probably be a wider "spaceselector" that can be shared by multiple actions. So didn't want to do that know. Getting spaceid for datafiles is quite is too since it's in the url.